.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+##### Description
+
+<!---
+Please include a short description of what your PR does and / or the motivation behind it
+--->
+
+##### Checklist
+
+- [ ] Built with `cargo build`
+- [ ] Built with `nix build`
+- [ ] Formatted with `cargo fmt`
+<!-- - [ ] Linted with `cargo clippy` -->
+- [ ] Ran tests with `cargo test`
+- [ ] Ran flake checks with `nix flake check`
+- [ ] Added or updated relevant tests (leave unchecked if not applicable)
+- [ ] Added or updated relevant documentation (leave unchecked if not applicable)
+
+##### Validating with `install.determinate.systems`
+
+Once a maintainer has added the `upload to s3` label to this PR, it will become available for installation via `install.determinate.systems`:
```

Comment:
```suggestion
If a maintainer has added the `upload to s3` label to this PR, it will become available for installation via `install.determinate.systems`:
```